addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -114,6 +114,17 @@ const FormEditor = options.Class.extend({
     _getMark: function () {
         return this.$target[0].dataset.mark;
     },
+    /**
+     * Replace all `"` character by `&quot;` and all `'` character by `&apos;`.
+     * This is needed in order to be able to perform querySelector of this
+     * type: `querySelector(`[name="${name}"]`)`.
+     *
+     * @param {HTMLElement} fieldEl
+     */
+    _getNameWithoutQuotes(name) {
```

Comment:
```suggestion
    _getQuotesEncodedName(name) {
```
So you know what the method (kinda) does just by reading the name, otherwise it looks like it's removing it